website/static/js/pages/base-page.js

```diff
@@ -87,6 +87,16 @@ $(document).on('click', '.project-toggle', function() {
     return false;
 });
 
+$(document).ready(function () {
```

Comment:
There is already "document-ready" handler below this on ~L100. This code can go in that handler.